src/Interpreters/ThreadStatusExt.cpp

```diff
@@ -40,13 +40,39 @@ namespace ErrorCodes
     extern const int CANNOT_SET_THREAD_PRIORITY;
 }
 
+const std::vector<UInt64> ThreadGroupStatus::getInvolvedThreadIds() const
```

Comment:
This const looks excessive
```suggestion
std::vector<UInt64> ThreadGroupStatus::getInvolvedThreadIds() const
```